src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.doWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
+        truck.stopWork();
```

Comment:
here better to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop